### PR TITLE
5051 fix sqlite default value

### PR DESCRIPTION
**Pull request by jasonku**
Addresses #5051

### CODE REVIEW

File: test/integration/query-interface.test.js [hunk 1]
```diff
@@ -177,7 +182,20 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
           }
           expect(username.type).to.equal(assertVal);
           expect(username.allowNull).to.be.true;
-          expect(username.defaultValue).to.be.null;
+
+          switch (dialect) {
+            case 'sqlite':
+              expect(username.defaultValue).to.be.undefined;
```
Comment (mickhansen) [inline test/integration/query-interface.test.js]: Should it not be undefined for all dialects?

File: test/integration/query-interface.test.js [hunk 2]
```diff
@@ -177,7 +182,20 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
...
+              break;
+            default:
+              expect(username.defaultValue).to.be.null;
+          }
+
+          switch (dialect) {
+            case 'sqlite':
+              expect(city.defaultValue).to.be.null;
```
Comment (mickhansen) [inline test/integration/query-interface.test.js]: Should it not be null for all dialects?

File: test/integration/query-interface.test.js [hunk 3]
```diff
@@ -190,7 +208,13 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
           }
           expect(isAdmin.type).to.equal(assertVal);
           expect(isAdmin.allowNull).to.be.true;
-          expect(isAdmin.defaultValue).to.be.null;
+          switch (dialect) {
+            case 'sqlite':
+              expect(isAdmin.defaultValue).to.be.undefined;
```
Comment (mickhansen) [inline test/integration/query-interface.test.js]: Should it not be undefined for all dialects?

Comment (jasonku) [inline test/integration/query-interface.test.js]: Should it be? I haven't explored this behavior with other dialects and I don't have tests set up to run other dialects' tests, so I don't really feel comfortable making changes for other dialects other than sqlite.

Comment (mickhansen) [inline test/integration/query-interface.test.js]: I think it should, no dialects should report a null default value if there in fact is no default value, null is special.
I'll merge your sqlite fix for now in any case.

**Pull request merged**